src/test/scala/org/apache/mesos/chronos/schedule/CronParserSpec.scala

```diff
@@ -22,4 +24,13 @@ class CronParserSpec extends SpecificationWithJUnit{
      result must beNone
    }
  }
+}
+
+class CronParserAltTZSpec extends SpecificationWithJUnit with AlternativeTimezoneTest {
+  "In US/Pacific Cron Parser should" in {
+   "Return the schedule in UTC" in {
+     val result = CronParser("0 18 * * *")
+     result.get.start.hourOfDay().get must_== 18
```

Comment:
So this means that cron expressions are parsed in the context of 'the current timezone'. Which I guess is right (well, more correct than the other option).